src/System.Drawing.Common/tests/BrushesTests.cs

```diff
@@ -0,0 +1,190 @@
+            yield return Brush(() => Brushes.PaleGoldenrod, Color.PaleGoldenrod);
+            yield return Brush(() => Brushes.PaleGreen, Color.PaleGreen);
+            yield return Brush(() => Brushes.PaleTurquoise, Color.PaleTurquoise);
+            yield return Brush(() => Brushes.PaleVioletRed, Color.PaleVioletRed);
+            yield return Brush(() => Brushes.PapayaWhip, Color.PapayaWhip);
+            yield return Brush(() => Brushes.PeachPuff, Color.PeachPuff);
+            yield return Brush(() => Brushes.Peru, Color.Peru);
+            yield return Brush(() => Brushes.Pink, Color.Pink);
+            yield return Brush(() => Brushes.Plum, Color.Plum);
+            yield return Brush(() => Brushes.PowderBlue, Color.PowderBlue);
+            yield return Brush(() => Brushes.Purple, Color.Purple);
+            yield return Brush(() => Brushes.Red, Color.Red);
+            yield return Brush(() => Brushes.RosyBrown, Color.RosyBrown);
+            yield return Brush(() => Brushes.RoyalBlue, Color.RoyalBlue);
+            yield return Brush(() => Brushes.SaddleBrown, Color.SaddleBrown);
+            yield return Brush(() => Brushes.Salmon, Color.Salmon);
+            yield return Brush(() => Brushes.SandyBrown, Color.SandyBrown);
+            yield return Brush(() => Brushes.SeaGreen, Color.SeaGreen);
+            yield return Brush(() => Brushes.SeaShell, Color.SeaShell);
+            yield return Brush(() => Brushes.Sienna, Color.Sienna);
+            yield return Brush(() => Brushes.Silver, Color.Silver);
+            yield return Brush(() => Brushes.SkyBlue, Color.SkyBlue);
+            yield return Brush(() => Brushes.SlateBlue, Color.SlateBlue);
+            yield return Brush(() => Brushes.SlateGray, Color.SlateGray);
+            yield return Brush(() => Brushes.Snow, Color.Snow);
+            yield return Brush(() => Brushes.SpringGreen, Color.SpringGreen);
+            yield return Brush(() => Brushes.SteelBlue, Color.SteelBlue);
+            yield return Brush(() => Brushes.Tan, Color.Tan);
+            yield return Brush(() => Brushes.Teal, Color.Teal);
+            yield return Brush(() => Brushes.Thistle, Color.Thistle);
+            yield return Brush(() => Brushes.Tomato, Color.Tomato);
+            yield return Brush(() => Brushes.Transparent, Color.Transparent);
+            yield return Brush(() => Brushes.Turquoise, Color.Turquoise);
+            yield return Brush(() => Brushes.Violet, Color.Violet);
+            yield return Brush(() => Brushes.Wheat, Color.Wheat);
+            yield return Brush(() => Brushes.White, Color.White);
+            yield return Brush(() => Brushes.WhiteSmoke, Color.WhiteSmoke);
+            yield return Brush(() => Brushes.Yellow, Color.Yellow);
+            yield return Brush(() => Brushes.YellowGreen, Color.YellowGreen);
+        }
+
+        public static object[] Brush(Func<Brush> brushThunk, Color expectedColor) => new object[] { brushThunk, expectedColor };
```

Comment:
This thunk is necessary because we want to assert:

```cs
Assert.Same(Brushes.White, Brushes.White);
```

but to make the test shorter (to use xunit) this is necessary